pyodide_build/config.py

```diff
@@ -228,7 +230,8 @@ def _get_make_environment_vars(self) -> Mapping[str, str]:
     "rustflags": "-C link-arg=-sSIDE_MODULE=2 -C link-arg=-sWASM_BIGINT -Z link-native-libraries=no",
     "cargo_build_target": "wasm32-unknown-emscripten",
     "cargo_target_wasm32_unknown_emscripten_linker": "emcc",
-    "rust_toolchain": "nightly-2024-01-29",
+    "rust_toolchain": "nightly-2025-01-15",
```

Comment:
Let's mention the default rust toolchain update in the changelog.

Reply:
This change isn't really needed to make this work. Maybe I can revert it here and put it by itself in a separate pr as a follow-up.